apps/sms/views/conversation/js/conversation.js

```diff
@@ -440,6 +440,8 @@ var ConversationView = {
   },
 
   getIdIterator: function conv_getIdIterator() {
+    // TODO: Will need to replace with Conversation service method or caching
+    // the ids locally.
     return Threads.active.messages.keys();
```

Comment:
The reason why I don't use this.activeThread now is because we it might be forgotten easily once the service is ready and we do cache the this.activeThread in the view. No error will be displayed if we don't use the service method to change conversation properties. But yes we can use this.activeThread right now with no problem and I'm fine if you think leaving the TODO comments here is enough.